src/special.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
```

Comment:
My intention is that it will be similar to [this](http://docs.scipy.org/doc/scipy/reference/special.html) or [these](http://docs.julialang.org/en/release-0.4/manual/mathematical-operations/#special-functions). There are other things we can move into here, but I'll do as a separate pr.

This name is so familiar to me I'm not sure how to improve it, what would work for you? Would putting this and `statistics.js` in a `math` folder help?